### PR TITLE
Build release version of Microsoft.DiaSymReader[.PortablePdb]

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
Changes missing from #11945.

**Comment by cston** [comment]
@tmat please review.
@mattgertz for approval.

**Comment by tmat** [comment]
:+1:

File: src/Dependencies/Microsoft.DiaSymReader/Version.targets [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <AssemblyVersion>1.0.8</AssemblyVersion>
-    <IsReleaseVersion>false</IsReleaseVersion>
+    <IsReleaseVersion>true</IsReleaseVersion>
```
Comment (tmat) [inline src/Dependencies/Microsoft.DiaSymReader/Version.targets]: This is dead code, but no harm done.

**Comment by cston** [comment]
@tmat Additional fixes.

File: build/BuildNuGets.csx [hunk 2]
```diff
@@ -207,7 +207,7 @@ void GeneratePublishingConfig()
     {
         // nuget:
         var packages = MakeRoslynPackageElements(isRelease: true).Concat(MakePackageElementsForPublishedDependencies(isRelease: true));
-        GeneratePublishingConfig("nuget_org-packages.config", packages);
+        GeneratePublishingConfig("myget_org-packages.config", packages);
```
Comment (tmat) [inline build/BuildNuGets.csx]: This is odd. Why not publish to nuget?

**Pull request merged**